docs/migration-recipe.md

```diff
@@ -0,0 +1,117 @@
+Migrating to new host:
+----------------------
+
+### Prepare salt configuration for migration:  
+
+1.  Ensure that salt-master, loadbalancer, and host in question are up, and that their health check for the relevant service is failing in haproxy after the host is fully up
```

Comment:
Could add bulleted list of commands to run:

```
laptop:psf-salt user$ vagrant up salt-master
laptop:psf-salt user$ vagrant up loadbalancer
laptop:psf-salt user$ vagrant up host
```

we might also want to make it explicit that we are doing this locally.  so instead of "Ensure that salt-master, loadbalancer, and host in question are up" Maybe "Ensure that salt-master, loadbalancer, and host in question can be brought up with vagrant locally"